src/vs/workbench/services/runtimeSession/common/runtimeSession.ts

```diff
@@ -2417,6 +2417,32 @@ export class RuntimeSessionService extends Disposable implements IRuntimeSession
 		}
 	}
 
+	/**
+	 * Resolves a path by expanding tildes and resolving symlinks.
+	 *
+	 * @param path The path to resolve
+	 * @returns The resolved path with tildes expanded and symlinks resolved
+	 */
+	private async resolvePath(path: string): Promise<string> {
+		const userHome = await this._pathService.userHome();
+		const userHomePath = userHome.scheme === Schemas.file ? userHome.fsPath : userHome.path;
+
+		// First expand tildes
+		const untildifiedPath = untildify(path, userHomePath);
```

Comment:
Would `ExtUri.resolvePath` here help for potentially relative paths? (Maybe we dont need to worry about that though?)

Reply:
I think neither of these paths can be relative (at least for now) so hopefully it's okay.